Worksheet_C/READ.ME.md

```diff
@@ -0,0 +1,9 @@
+The algorithm checks both list I and J and then will check whether I is equal to J and if list [i] is equal to [j]. If they are the same, the list contains a duplicate and will return as true. If there isn’t a duplicate, it will return as being false.
+2)	Quadratic will have the slowest running time because it will quadruple the time required to run the script.
+3)	.
+4)	.
+5)	.
+6)	O (log in)
+7)	O(n)
+8)	The faster algorithm to run would be O (n log n) as O(n) is linear. This means that the function is proportional to the size of the list. O (n log n) will take only a few iterations as it halves the list whereas O (n) checks each item against each other. 
```

Comment:
O(n log n) is slower than O(n) for large n, however this is irrelevant as the first algorithm is O(n^2).